homepage.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
The meta tags here are good, did you pull them from the bootstrap docs pages? I will have to re-learn what these do and how they function.